src/main/java/com/mraof/minestuck/data/DataMapGenerator.java

```diff
@@ -0,0 +1,355 @@
+		
+		fuels.add(MSTags.Items.DEAD_LOGS, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_DOOR, new FurnaceFuel(200), false);
+		fuels.add(MSItems.DEAD_TRAPDOOR, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_FENCE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_FENCE_GATE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_STAIRS, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_SLAB, new FurnaceFuel(150), false);
+		fuels.add(MSItems.DEAD_BUTTON, new FurnaceFuel(100), false);
+		fuels.add(MSItems.DEAD_PRESSURE_PLATE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_BOOKSHELF, new FurnaceFuel(300), false);
+		fuels.add(MSItems.DEAD_LADDER, new FurnaceFuel(300), false);
+		
+		fuels.add(MSTags.Items.CINDERED_LOGS, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_DOOR, new FurnaceFuel(200), false);
+		fuels.add(MSItems.CINDERED_TRAPDOOR, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_FENCE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_FENCE_GATE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_STAIRS, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_SLAB, new FurnaceFuel(150), false);
+		fuels.add(MSItems.CINDERED_BUTTON, new FurnaceFuel(100), false);
+		fuels.add(MSItems.CINDERED_PRESSURE_PLATE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_BOOKSHELF, new FurnaceFuel(300), false);
+		fuels.add(MSItems.CINDERED_LADDER, new FurnaceFuel(300), false);
+		
+		fuels.add(MSTags.Items.PETRIFIED_LOGS, new FurnaceFuel(300), false);
+		
+		fuels.add(MSTags.Items.SHADEWOOD_LOGS, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_DOOR, new FurnaceFuel(200), false);
+		fuels.add(MSItems.SHADEWOOD_TRAPDOOR, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_FENCE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_FENCE_GATE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_STAIRS, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_SLAB, new FurnaceFuel(150), false);
+		fuels.add(MSItems.SHADEWOOD_BUTTON, new FurnaceFuel(100), false);
+		fuels.add(MSItems.SHADEWOOD_PRESSURE_PLATE, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_BOOKSHELF, new FurnaceFuel(300), false);
+		fuels.add(MSItems.SHADEWOOD_LADDER, new FurnaceFuel(300), false);
+		
+		/* PARROT IMITATIONS */
+		
+		var parrot = this.builder(NeoForgeDataMaps.PARROT_IMITATIONS);
+		parrot.add((Holder<EntityType<?>>) MSEntityTypes.IMP, new ParrotImitation(MSSoundEvents.ENTITY_IMP_AMBIENT.get()), false);
+		parrot.add((Holder<EntityType<?>>) MSEntityTypes.OGRE, new ParrotImitation(MSSoundEvents.ENTITY_OGRE_AMBIENT.get()), false);
+		parrot.add((Holder<EntityType<?>>) MSEntityTypes.BASILISK, new ParrotImitation(MSSoundEvents.ENTITY_BASILISK_AMBIENT.get()), false);
+		parrot.add((Holder<EntityType<?>>) MSEntityTypes.LICH, new ParrotImitation(MSSoundEvents.ENTITY_LICH_AMBIENT.get()), false);
```

Comment:
Is the plan here to add support for resource packs or potential future implementation? Parrots do not imitate the mobs as is due to there not being a relevant sound file

Reply:
I tested again just now and the parrots do correctly imitate the mobs, using the existing sound events, as they should. Ogres and basilisk IMO are on the quieter end, but do work. 

The primary use case is the imps, since parrots are most likely to be at someone's base, where imps will spawn when they enter, but I think it's good practice to have the other three ready for the niche cases.